From 952587c953a9fb873210b2be7abb0d17dfe95fa9 Mon Sep 17 00:00:00 2001
From: Peter Chiu <chui-hao.chiu@mediatek.com>
Date: Thu, 13 Aug 2026 10:07:12 +0800
Subject: [PATCH] mtk: mac80211: Fix jiffies conversion macro guard

Fix the conditional compilation guard for the secs_to_jiffies macro to
ensure it is defined only when not already present, preventing
compile errors due to upstream upgrade.

Feature:
Stability

Signed-off-by: Peter Chiu <chui-hao.chiu@mediatek.com>
---
 backport-include/linux/jiffies.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/backport-include/linux/jiffies.h b/backport-include/linux/jiffies.h
index 002a5571..cf8f2527 100644
--- a/backport-include/linux/jiffies.h
+++ b/backport-include/linux/jiffies.h
@@ -3,7 +3,7 @@
 #include_next <linux/jiffies.h>
 #include <linux/version.h>
 
-#if LINUX_VERSION_IS_LESS(6,12,0)
+#ifndef secs_to_jiffies
 
 /**
  * secs_to_jiffies: - convert seconds to jiffies
@@ -18,6 +18,6 @@
  */
 #define secs_to_jiffies(_secs) ((_secs) * HZ)
 
-#endif /* < 6.12.0 */
+#endif /* secs_to_jiffies */
 
 #endif /* __BACKPORT_JIFFIES_H */
-- 
2.45.2

